src/main/java/Task.java

```diff
@@ -0,0 +1,45 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    protected static final DateTimeFormatter dtf = 
+        DateTimeFormatter.ofPattern("dd/MM/uuuu HHmm");
+
+    public Task(boolean done, String description) {
```

Comment:
Considering that Task objects should be instantiated as one of its subclasses (ToDo, Event, Deadline), perhaps this class should be declared Abstract